src/auth/models/strategy.class.ts

```diff
@@ -182,6 +186,7 @@ export abstract class Strategy extends events.EventEmitter {
         this.initializePassport()
         this.initializeSession()
         this.initializeKeepAlive()
+        this.initialiseCSRF()
```

Comment:
Question - because the options are already can we add the check  if (this.options.useCSRF) here itself.